AdminUi/src/AdminUi/ClientApp/src/app/components/quotas/identity/identity-edit/identity-edit.component.ts

```diff
@@ -99,6 +126,32 @@ export class IdentityEditComponent {
         }
     }
 
+    saveIdentity(): void {
+        if (this.tier && this.tier.id) {
+            this.identityService.updateIdentityTier(this.identity, this.tier).subscribe({
+                next: () => {
+                    this.snackBar.open("Identity updated successfully. Reloading...", "Dismiss", {
+                        verticalPosition: "top",
+                        horizontalPosition: "center"
+                    });
+
+                    this.loading = true;
+                    setTimeout(() => {
+                        this.getIdentity();
+                    }, 2000);
+                },
+                error: (err: any) => {
+                    this.loading = false;
```

Comment:
When is the loading animation turned on?
I see it for the waiting mechanism while it's "reloading" but not before the initial request.
Shouldn't we have `this.loading = true` as the first statement in the current method?